tests/test_selenium_app.py

```diff
@@ -0,0 +1,136 @@
+import os
+import random
+from threading import Thread
+
+import pytest
+from dotenv import find_dotenv, load_dotenv
+from selenium import webdriver
+
+import app
+import common
+from trello_request import TrelloRequest
+
+
+class TrelloCreateBoard(TrelloRequest):
```

Comment:
These classes that essentially wrap a single function are fairly limited - a more common pattern might have us combine this with the TrelloDeleteBoard class into a TrelloBoardService that has the two separate methods, and we could do similarly with cards for example